app/src/main/java/org/oppia/app/topic/lessons/TopicLessonsFragment.kt

```diff
@@ -16,6 +16,23 @@ private const val KEY_CURRENT_EXPANDED_LIST_INDEX = "CURRENT_EXPANDED_LIST_INDEX
 
 /** Fragment that contains subtopic list for lessons mode. */
 class TopicLessonsFragment : InjectableFragment(), ExpandedChapterListIndexListener, StorySummarySelector {
+
+  companion object {
+    /** Returns a new [TopicLessonsFragment] */
```

Comment:
Add full-stop at the end of the sentence `/** Returns a new [TopicLessonsFragment]. */`

Reply:
Done